compose/snippets/src/main/java/com/example/compose/snippets/modifiers/CustomModifierSnippets.kt

```diff
@@ -259,14 +260,15 @@ class ScrollableNode :
 object CustomModifierSnippets14 {
     // [START android_compose_custom_modifiers_14]
     class CircleNode(var color: Color) : Modifier.Node(), DrawModifierNode {
-        private val alpha = Animatable(1f)
+        private lateinit var alpha: Animatable<Float, AnimationVector1D>
 
         override fun ContentDrawScope.draw() {
             drawCircle(color = color, alpha = alpha.value)
             drawContent()
         }
 
         override fun onAttach() {
+            alpha = Animatable(1f)
```

Comment:
I wonder whether it'd be more apropriate if we keep this 
`private var alpha = Animatable(1f)` but override `onReset()` and override with a new value. 
If someone at some poitn touches the `alpha` before `onAttach`, it won't crash the app.

Reply:
Yeah, that could work too. For a simple/tiny example like this, it probably doesn't matter which way it's handled, but I could see that being a good option if the class is likely to grow and the lifecycle becomes harder to reason about. I'll leave it up to you fine folks to figure out which you prefer since the DAC docs probably have to get updated to explain either option and I don't have permissions to merge anyway 😆

Reply:
Added a note on DAC. Thanks Ian! ❤️